### DISCUSSION TITLE
Punching and Pointing

### DISCUSSION THREAD

**Discussion by mjsonic77** [FakeFishGames/Barotrauma#6906] [Ideas and suggestions]
Allow the ability to punch and point when you have nothing in your hands.
Hold right click to point. (Or hold your fists up like a boxer.)
Then left click to punch.

**Comment by StruggleDuck** [comment]
this sounds like a good idea, besides the fact that it would be pretty much useless due to attacks with wrenches and screwdrivers already doing practically no damage. Still think it should be added as some kind of last resort against whatever happens to be attacking you

**Reply by mLuby** [reply]
Good idea. How about [hold aim] to fists up like a boxer, [press fire] to punch, ending with pointer finger extended. [hold fire] keeps the pointer finger extended. [press fire] again to punch/point with the other arm so you can alternate punches back and forth.

**Reply by Hazmat616** [reply]
> Good idea. How about [hold aim] to fists up like a boxer, [press fire] to punch, ending with pointer finger extended. [hold fire] keeps the pointer finger extended. [press fire] again to punch/point with the other arm so you can alternate punches back and forth.

Captain look a mudraptor behind you! Punches him in the face.

**Comment by Hazmat616** [comment]
If griefing is a concern make it a multiplier setting and be disabled by space herpes.

**Comment by jigardi** [comment]
Have my upvote!
This is a great idea, especially since empty hands currently do nothing.
It would also allow devs or modders to add various events that make the world feel more lived in:

_People getting robbed, drunks starting fights, illegal boxing tournaments._
